R/utils.R

```diff
@@ -57,3 +58,41 @@ modify_vector <- function(x, y = NULL) {
   lnames <- function(x) tolower(names(x))
   c(x[!(lnames(x) %in% lnames(y))], y)
 }
+
+#' Generate Bearer auth token
+#'
+#' This function allows to generate new auth token base on client id and secret
+#' If you looking for more infromation see: \url{https://auth0.com/docs/api/management/v2/tokens}
+#' @param client_id The OAuth application id
+#' @param client_secret The OAuth application secret
+#' @param .domain By default \code{\link{api_domain}}
+#' @return Auth0 Bearer token
+#' @export
+generate_token <- function(client_id, client_secret, .domain = api_domain()) {
+  resp <-
+    httr::POST(
```

Comment:
what happens on failed request (like connectivity issue?) Are we handling that?

Reply:
This is the correct behavior. The function just returns an error and stop the execution.